pytorch_lightning/accelerators/ddp_torchelastic_accelerator.py

```diff
@@ -110,9 +110,6 @@ def ddp_train(self, process_idx, model):
             process_idx:
             mp_queue: multiprocessing queue
             model:
-
-        Returns:
```

Comment:
It just looks like these docstrings are outdated. It seems the method does in fact return a result.

Reply:
@akihironitta mind revisit there you edited...

Reply:
@awaelchli Thank you for your review! You're right. It actually returns `results`, but I have no idea what explanation to add. Can you maybe help?